src/main/java/com/wolox/training/repositories/BookRepositiry.java

```diff
@@ -0,0 +1,8 @@
+package com.wolox.training.repositories;
+
+import com.wolox.training.models.Book;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface BookRepositiry extends JpaRepository<Book, String> {
```

Comment:
This is what we talked about :) It should be `extends JpaRepository<Book, Long>`, because the book id is Long :)

Reply:
Yes! sorry the change is present in the following PR :)

Reply:
Great then!